### DISCUSSION TITLE
Should use bigint or number for counter?

### DISCUSSION THREAD

**Discussion by sergiocarneiro** [MasterKale/SimpleWebAuthn#453] [Troubleshooting]
In the [server documentation](https://simplewebauthn.dev/docs/packages/server#additional-data-structures) it says:
> Consider `BIGINT` since some authenticators return atomic timestamps as counters

However `number` is used instead throughout the library.

Is anything outdated?

**Comment by MasterKale** [comment]
I should probably update that part of the docs. I haven't seen an authenticator return an atomic timestamp for `counter` since _maybe_ 2021. macOS Chrome's internal authenticator used to do this, and was the only one I remember seeing, back in 2019 when I first discovered all this, but it simply returns 0 now so a simpler data type should suffice.
